IdentityCore/src/cache/crypto/MSIDAssymetricKeyLookupAttributes.m

```diff
@@ -25,46 +25,41 @@
 
 @implementation MSIDAssymetricKeyLookupAttributes
 
+/*
+ This particular query generates the asymmetric key pair but only saves the private key in the keychain.
+ Please refer to https://developer.apple.com/documentation/security/certificate_key_and_trust_services/keys/generating_new_cryptographic_keys?language=objc
+ */
 - (NSDictionary *)assymetricKeyPairAttributes
 {
     NSMutableDictionary *keyPairAttr = [NSMutableDictionary new];
-    keyPairAttr[(__bridge id)kSecAttrIsPermanent] = @YES;
     keyPairAttr[(__bridge id)kSecAttrKeyType] = (__bridge id)kSecAttrKeyTypeRSA;
     keyPairAttr[(__bridge id)kSecAttrKeySizeInBits] = @2048;
     keyPairAttr[(__bridge id)kSecAttrLabel] = self.keyDisplayableLabel;
-    keyPairAttr[(__bridge id)kSecAttrAccessible] = (__bridge id)kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly;
     
     NSMutableDictionary *privateKeyAttr = [NSMutableDictionary new];
+    privateKeyAttr[(__bridge id)kSecAttrIsPermanent] = @YES;
```

Comment:
Previous code added kSecAttrAccessible and kSecAttrIsPermanent to the keyPairAttr. New code puts it to privateKeyAttr. Which one is correct?